src/services/auth.js

```diff
@@ -1,9 +1,7 @@
 import request from '../utils/request';
 
-const { endpoints } = window;
```

Comment:
I'm curious why you removed the local unpacking of `endpoints` from `window` throughout; this isn't really a functional change, and seems to just complicate the PR with non-essential edits. (I don't know whether the destructing actually *copies* `window.endpoints` to the local `endpoints` or just constructs a reference: in the former case, your change might be a bit more efficient... but I would hope that it's just a mapped reference without moving any data.)